compiler/grainformat/reformat.re

```diff
@@ -2962,7 +2987,7 @@ let data_print =
       original_source: array(string),
     ) => {
   Doc.join(
-    Doc.comma,
+    Doc.concat([Doc.comma, Doc.hardLine, Doc.hardLine]),
```

Comment:
Why 2 hardlines here?

Reply:
I think they look better with a clear line between them, as in your failing example. Just a style choice really, but they seem squashed with no blank line between them

Reply:
Also, because the formatter usually adds a line between top level statements but these are seen as the same top level statement in the AST

Reply:
We might want to discuss with @ospencer and @peblair. I think it might be common for these to only have 1 newline between, but it'd be good to get more opinions. e.g.
```grain
enum Foo { Baz },
record Bar { a: Foo }
```

Reply:
I would personally vote for only one line, since in my mind I think of them as the same statement. The extra line adds some dissonance for me. Makes me wonder if we should go back to the `and` keyword, much more clear...

Reply:
I'll set it to one line with no gap, but also a vote for and!